README.md

```diff
@@ -57,7 +62,7 @@ of hosted Kubernetes environments and whether `istio-cni` has been trialed in th
    1. `kubectl create clusterrolebinding cni-cluster-admin-binding --clusterrole=cluster-admin --user=istio-user@gmail.com`
       1. User `istio-user@gmail.com` is an admin user associated with the gcloud GKE cluster
 
-1. Install Istio via Helm including these options `--set istio_cni.enabled=true --set istio-cni.cniBinDir=/home/kubernetes/bin`
+1. Create the Istio CNI manifests with this option `--set cniBinDir=/home/kubernetes/bin`
```

Comment:
Maybe add "with this Helm chart option" or "In the Helm command, create the ..."

Reply:
will fix